src/dlr_common.cc

```diff
@@ -50,15 +50,31 @@ std::string dlr::GetBasename(const std::string& path) {
 void dlr::ListDir(const std::string& dirname, std::vector<std::string>& paths) {
   dmlc::io::URI uri(dirname.c_str());
```

Comment:
change the variable name since its no longer just a directory name

Reply:
changed dirname to path